assembly_and_binning/tutorial_assembly_and_binning.md

```diff
@@ -0,0 +1,634 @@
+conda config --add channels conda-forge
+```
+
+## Getting started with ATLAS
+
+We'll organize ourselves as if we're starting a brand new project. 
+
+```
+cd ~
+mkdir atlas_workflow
+cd atlas_workflow/
+```
+
+We also need to install atlas to get this metagenome party started. We'll do that with this code, and we'll go over what it all means in our evening session.
+
+```
+mamba create -y -n atlas metagenome-atlas=2.9.1
+conda activate atlas
+```
+
+Ok! We now have atlas installed. 
+Let's checked that our installation worked by running atlas.
+
+```
+atlas
+```
+
+You should see a help message that looks like this:
+
+```
+Usage: atlas [OPTIONS] COMMAND [ARGS]...
+
+  ATLAS - workflows for assembly, annotation, and genomic binning of
+  metagenomic and metatranscriptomic data.
+
+  For updates and reporting issues, see: https://github.com/metagenome-
+  atlas/atlas
+
+Options:
+  --version   Show the version and exit.
+  -h, --help  Show this message and exit.
+
+Commands:
+  download     download reference files (need ~50GB)
+  init         prepare configuration file and sample table for atlas run
+  init-public  Prepare atlas run from public data from SRA
+  run          run atlas main workflow
+```
+
+## Retrieving the data
+
+We will be working with two metagenomes from Woods Hole!
+The samples come from the Trunk River lagoon in Falmouth (just a short ways up the road!) which has large amounts of biomass and which experiences periodic bright yellow microbial blooms, typically associated with disturbances to the ecosystem ([source](https://environmentalmicrobiome.biomedcentral.com/articles/10.1186/s40793-019-0348-0?_ga=2.190106795.160855720.1658156183-1639102303.1656704578)).
+These metagenomes were sampled from human-made disturbances that caused blooms.
+The accession numbers are SRR8859675 and SRR8859678. 
+Each sample has about ~1 GB of data, and it would take too long to run atlas on the entire dataset (~4 hours).
+
+For this tutorial, we subsampled the SRR8859675 to the first 1 million paired-end reads, and we will be working with this subsampled data. 
+
+First, let's download our data.
+
+```
+mkdir data
+wget -O data/SRR8859675_R1.fq.gz https://osf.io/mhr9y/download
+wget -O data/SRR8859675_R2.fq.gz https://osf.io/8usdv/download 
+```
+
+We can check that the download was successful by listing the files with `ls`:
+
+```
+ls -lh data
+```
+
+We see that we now have about 160 MB of data.
+
+
+```
+-rw-rw-r-- 1 stamps2022 stamps2022 40M Jul 24 18:22 SRR8859675_R1.fq.gz
+-rw-rw-r-- 1 stamps2022 stamps2022 49M Jul 24 18:22 SRR8859675_R2.fq.gz
+```
+
+## Configuring atlas
+
+We now need to tell atlas where our data is. 
+We do this with the `atlas init` command. 
+We'll also tell atlas that we're working with metagenome data, and that we want atlas to do assembly with megahit.
+(you could definitely use a different assembler! We're using megahit here because it uses the least ram and takes the least amount of time.)
```

Comment:
```suggestion
We'll also tell atlas that we're working with metagenome data, and that we want atlas to do assembly with megahit.
You could definitely use a different assembler! We're using megahit here because it uses the least RAM and takes the least amount of time.
```